src/transport_ofi.c

```diff
@@ -1056,6 +1062,9 @@ static int shmem_transport_ofi_ctx_init(shmem_transport_ctx_t *ctx, int id)
     OFI_CHECK_RETURN_MSG(ret, "get_cntr creation failed (%s)\n", fi_strerror(errno));
 
     ret = fi_cq_open(shmem_transport_ofi_domainfd, &cq_attr, &ctx->cq, NULL);
+    if (ret && errno == FI_EMFILE) {
+        DEBUG_STR("Consider increasing the open file limit with 'ulimit' command");
```

Comment:
Need more context in the debug message, e.g.:
```Context creation failed because of open files limit, consider increasing with 'ulimit' command```.

Reply:
We may want to consider updating the "CHECK_RETURN_MSG" output since these are no longer considered to be errors.  Not sure I have a great suggestion, just an observation.